core/src/main/java/com/linecorp/armeria/client/ClientRequestContext.java

```diff
@@ -161,6 +165,35 @@ static ClientRequestContextBuilder builder(RpcRequest request, URI uri) {
         return new ClientRequestContextBuilder(request, uri);
     }
 
+    /**
+     * Returns the {@link ServiceRequestContext} whose {@link Service} invokes the {@link Client}
+     * {@link Request} which created this {@link ClientRequestContext}, or {@code null} if this client request
+     * was not made in the context of a server request.
+     */
+    @Nullable
+    ServiceRequestContext serviceRequestContext();
+
+    /**
+     * {@inheritDoc} If {@link #serviceRequestContext()} is not {@code null} and the
+     * {@link ServiceRequestContext} has the {@link Attribute}, this will return the {@link Attribute}.
```

Comment:
I think you also need to say 'if the attribute is not in this context'? Perhaps better rephrasing to express the exact condition.

Reply:
Added Thanks!